pkgs/applications/video/lossless-cut/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchFromGitHub, mkYarnPackage }:
+
+mkYarnPackage rec {
+  name = "lossless-cut";
+  version = "3.23.7";
+
+  src = fetchFromGitHub {
+    owner = "mifi";
+    repo = name;
+    rev = "v${version}";
+    sha256 = "14vfmidj0m11vd1asghmcxwj102c468n6wb4swagnjzrxqh22xa3";
+  };
+
+  yarnNix = ./yarn.nix;
+  yarnLock = "${src}/yarn.lock";
+  packageJSON = "${src}/package.json";
+
+  meta = with stdenv.lib; {
+    description = "The swiss army knife of lossless video/audio editing";
+    license = licenses.mit;
+    homepage = "https://mifi.no/losslesscut";
+    maintainers = with maintainers; [ ShamrockLee ];
```

Comment:
You need to add yourself to maintainers/maintainer-list.nix, preferably do that in a separate commit.

Reply:
The problem is that I added myself to the `maintainer-list.nix` in another new package commit waiting to be merged (i.e. #91270).
I don't know if it would cause conflicts.

Reply:
I just added myself to the `maintainer-list.nix` to make it evaluatable.